app/src/main/java/com/example/albumsapp/adapter/AlbumAdapter.kt

```diff
@@ -30,6 +32,8 @@ class AlbumAdapter(
 
         holder.button.setOnClickListener {
             onAlbumClicked(item)
+            val action = AlbumsListFragmentDirections.actionListOfAlbumsToListOfPhotos(item.id)
+            holder.itemView.findNavController().navigate(action)
```

Comment:
the idea behind `onAlbumClicked(item)` is to move `AlbumsListFragmentDirections` out. Please move these two lines into `onAlbumClicked` in fragment